src/cpp/include/coord.h

```diff
@@ -0,0 +1,51 @@
+#ifndef COORD_H
+#define COORD_H
```

Comment:
what is difference between coord.h and coordinator.h?

Reply:
Not used, left by mistake, removed.